addons/purchase_stock/models/purchase.py

```diff
@@ -316,6 +317,18 @@ def _compute_qty_received(self):
                 line._track_qty_received(total)
                 line.qty_received = total
 
+    @api.depends('product_uom_qty', 'date_planned')
+    def _compute_forecasted_issue(self):
+        for line in self:
+            warehouse = line.order_id.picking_type_id.warehouse_id
+            line.forecasted_issue = False
+            if line.product_id:
+                virtual_available = line.product_id.with_context(warehouse=warehouse.id, to_date=line.order_id.date_planned).read(['virtual_available'])[0]["virtual_available"]
```

Comment:
`to_date=line.order_id.date_planned`, date_planned of the order can be different than the one on the line. Also your compute depends of the one of the line then I assume that it should be `to_date=line.date_planned`